geo/src/main/java/org/odk/collect/geo/selection/SelectionSummarySheet.kt

```diff
@@ -73,10 +73,10 @@ internal class SelectionSummarySheet(context: Context, attrs: AttributeSet?) :
         }
 
         item.action?.let {
-            binding.action.text = item.action.text
+            binding.action.text = item.action!!.text
```

Comment:
This should use the block arg (or `it`) to avoid need the `!!`s here.

Reply:
Done.